.github/CODEOWNERS

```
@@ -0,0 +1,294 @@
+/openstack/sre                                         @artherd42 @fwiesel @rajivmucheli
+/openstack/swift                                       @sumitarora2786 @crenduchinta88 @majewsky @SuperSandro2000 @VoigtS
+/openstack/swift-drive-autopilot                       @sumitarora2786 @crenduchinta88 @majewsky @SuperSandro2000 @VoigtS
+/openstack/swift-utils                                 @sumitarora2786 @crenduchinta88 @majewsky @SuperSandro2000 @VoigtS
+/openstack/tempest/barbican-tempest                    @misamoylov @fwiesel @rajivmucheli
+/openstack/tempest/cinder-tempest                      @misamoylov @fwiesel
+/openstack/tempest/designate-tempest                   @misamoylov @fwiesel
+/openstack/tempest/glance-tempest                      @misamoylov @fwiesel @rajivmucheli
+/openstack/tempest/ironic-tempest                      @misamoylov @fwiesel
+/openstack/tempest/keystone-tempest                    @misamoylov @fwiesel @rajivmucheli
+/openstack/tempest/manila-tempest                      @kpawar-sap @misamoylov @fwiesel
+/openstack/tempest/neutron-tempest                     @misamoylov @fwiesel
+/openstack/tempest/nova-tempest                        @misamoylov @fwiesel
+/openstack/tempest/octavia-tempest                     @misamoylov @fwiesel
+/openstack/tempest/tempest-base                        @misamoylov @fwiesel @stefanhipfel
+/openstack/trust-manager                               @fwiesel
+/openstack/utils                                       @fwiesel @notandy @galkindmitrii @Carthaca @joker-at-work
+/openstack/vcenter-operator                            @joker-at-work @fwiesel @databus23
+/openstack/vcf                                         @chuan137
+/openstack/volume-claims                               @fwiesel
+
+/prometheus-exporters/apic-exporter                    @artherd42 @IvoGoman @geisslet
+/prometheus-exporters/arista-exporter                  @BerndKue @stefanhipfel
+/prometheus-exporters/blackbox-exporter                @defo89 @databus23
+/prometheus-exporters/cloudprober                      @artherd42 @defo89
+/prometheus-exporters/cronus-exporter                  @dhalimi1975 @LimorGargir @richardtief @geisslet
+/prometheus-exporters/cronus-openstack-alerts          @dhalimi1975
+/prometheus-exporters/cronus-reporter                  @dhalimi1975
+/prometheus-exporters/cronus-simulator                 @dhalimi1975
+/prometheus-exporters/cronus-updater                   @dhalimi1975
+/prometheus-exporters/documentation-prober             @dhalimi1975  # old
+/prometheus-exporters/event-exporter                   @jknipper @databus23
+/prometheus-exporters/hermes-query-exporter            @Kuckkuck @notque
+/prometheus-exporters/ipmi-exporter                    @stefanhipfel @BerndKue
+/prometheus-exporters/ipmi-service-discovery           @auhlig  # old
+/prometheus-exporters/ironic-exporter                  @BerndKue @stefanhipfel
```

```suggestion
/prometheus-exporters/ironic-exporter                  @BerndKue @stefanhipfel @sandzwerg 

```

So we have at least three